src/hooks/useMaintenanceBanner.ts

```diff
@@ -95,6 +95,7 @@ export function useMaintenanceBanner() {
     .with('UAT', () => t('titleTestEnv'))
     .with('DEV', () => '') // this environment has no maintenance banner
     .with('QA', () => '') // this environment has no maintenance banner
+    .with('DEV_REF', () => '') // this environment has no maintenance banner
     .exhaustive()
```

Comment:
I propose to change it into an `otherwise`. Leaving like this would potentially cause other crasshes if the STAGE env changes again. Plus I think that `null` value would be a better fit as the default here

 ```suggestion
    .otherwise(() => null)
```